Generator/Templates/GObject/class.sbntxt

```diff
@@ -38,65 +38,58 @@ namespace {{ namespace }}
     
     public partial class {{ name }} {{ write_inheritance }}
     {
-{{if !fundamental}}
+{{~ if !fundamental ~}}
         private static readonly TypeDescriptor GTypeDescriptor = TypeDescriptor.For("{{ type_name }}", {{ name }}.Native.get_type);
 
         #region Signals
         
-        {{- if signals_have_event_args }}
-        
+        {{~ if signals_have_event_args ~}}
         ///<summary>
         /// Indexer to connect with an SignalHandler<{{name}}>
         ///</summary>
         public SignalHandler<{{ name }}> this[Signal<{{ name }}> signal]
         {
             set => signal.Connect(this, value, true);
         }
-        {{ end }}
-
-        {{- for $signal in signals
-
-include 'signal.sbntxt' $signal '        ' name
-
-        end ~}}
+        {{~ end ~}}
+        {{~ for $signal in signals }}
+        {{ include 'signal.sbntxt' $signal name }}
+        {{ end ~}}
         
         #endregion
         
-        {{- if has_parent}} 
-        
+        {{~ if has_parent ~}} 
         #region Constructors
         
         protected internal {{ name }}(IntPtr ptr) : base(ptr) {}
         protected internal {{ name }}(params ConstructParameter[] properties) : base(properties) {}
         
         #endregion
         
-        {{- end }}
-{{- end }}
-
+        {{~ end ~}}
+{{~ end ~}}
         #region Methods
         
         protected static{{' new'|get_if_parent}} class Native
         {
-        {{~if !fundamental && (string.size get_type_function) > 0 
-            get_type_function | write_dll_import dll_import '            '
-        ~}}
+        
+            {{~if !fundamental && (string.size get_type_function) > 0 
+                '            ' + (get_type_function | get_dll_import dll_import)
+            ~}}
             public static extern ulong get_type();
-        {{ end ~}}
-
-            #region Properties
             
-            {{~ for $property in properties
-                include 'native_property.sbntxt' $property '            '
-            end ~}}
+            {{~ end ~}}
+            #region Properties
+            {{~ for $property in properties }}
+            {{ include 'native_property.sbntxt' $property ~}}
+            {{ end ~}}
             
             #endregion
         
             #region Methods
-            
-            {{~for $method in all_methods
-                include 'method.sbntxt' $method dll_import '            ' name
-            end ~}}
+            {{~ for $method in all_methods }}
+            {{ include 'method.sbntxt' $method dll_import name ~}}
+            {{ end ~}}
             
             #endregion
         }
```

Comment:
Need a line break here before the `#endregion`

Reply:
For me it is generated correctly? E.g.:

```C#
            [DllImport("libgtk-3.so.0", EntryPoint = "gtk_button_new_with_mnemonic")]
            public static extern IntPtr new_with_mnemonic(string label);
                        
            #endregion
```

Reply:
Done